src/dnx.coreclr.unix/dnx.coreclr.cpp

```diff
@@ -157,9 +198,9 @@ extern "C" HRESULT CallApplicationMain(PCALL_APPLICATION_MAIN_DATA data)
     }
 
     std::string coreClrDirectory;
-    void* coreClr = LoadCoreClr(coreClrDirectory);
+    LoadCoreClr(coreClrDirectory);
 
-    if (!coreClr)
+    if (!pLibCoreClr)
     {
         char* error = dlerror();
```

Comment:
Yes.  I will restructure this (as well as the return if the tpa can't be built).